echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/AbstractEventNotificationAgent.groovy

```diff
@@ -53,7 +53,7 @@ abstract class AbstractEventNotificationAgent implements EchoEventListener {
 
   @Override
   void processEvent(Event event) {
-    if (log.isDebugEnabled() && mapper != null) {
+    if (log.isDebugEnabled() && mapper != null && !event.getDetails().getType().equals("pubsub")) {
```

Comment:
Not worried about NPE here as the rest of the function assumes non-null type already